### ISSUE TITLE
Not generating grid

### ISSUE THREAD

**Issue by PedroMiguelSS** [ipfs/contributors#8]
Hi guys. I'm trying to use this package to build **Community** section on [`js.ipfs`](https://github.com/ipfs/js.ipfs.io) website. 
I did follow [usage](https://github.com/ipfs/contributors#usage) instructions but I was not able to succeed. Taken steps:

- `node scripts/bin/create js.ipfs.io --title "JS-IPFS Contributors" --org all --size 120`
- `make`
- `make serve`

After that, I was expecting that `localhost:1313` would render the hex grid with the contributors, but instead, it is showing an empty `<ul>` tag. 

I can see that images are generated (`public/images/js.ipfs.io`). But it's not displaying the grid correctly. 

Am I missing something here? @diasdavid @alanshaw

**Comment by alanshaw** [comment]
I'll take a look

**Comment by alanshaw** [comment]
@PedroMiguelSS PR for this is here: https://github.com/ipfs/contributors/pull/9 please could you verify everything works now?

TLDR; had to move a file to get new versions of hugo to build it

**Comment by PedroMiguelSS** [comment]
Yes, @alanshaw, it's working perfectly now. Thanks a lot!
